bin/setup

```diff
@@ -5,7 +5,7 @@ gem_root = Pathname.new(__dir__).join("..")
 
 unless gem_root.join("spec/manageiq").exist?
   puts "== Cloning manageiq sample app =="
-  system "git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 spec/manageiq"
+  system "git clone https://github.com/xlab-si/manageiq.git --branch remove-deprecated-vmdb-config --depth 1 spec/manageiq"
```

Comment:
@carbonin This needs to change before merge.  I had asked @tadeboro to do this to ensure this side is green.